aiida_cp2k/calculations/__init__.py

```diff
@@ -179,6 +179,11 @@ def define(cls, spec):
             "ERROR_GEOMETRY_CONVERGENCE_NOT_REACHED",
             message="The ionic minimization cycle did not converge for the given thresholds.",
         )
+        spec.exit_code(
+            501,
+            "ERROR_MAX_NUMBER_OPT_STEPS_REACHED",
```

Comment:
```suggestion
            "ERROR_MAXIMUM_NUMBER_OPTIMIZATION_STEPS_REACHED",
```

Let's use full words, if possible. We didn't use shortened words for the other error codes.

Reply:
I agree, I will add the changes